components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt

```diff
@@ -236,10 +237,12 @@ class DatabaseLoginsStorage(private val dbPath: String) : AutoCloseable, LoginsS
     @Throws(LoginsStorageException::class)
     override fun add(login: ServerPassword): String {
         return writeQueryCounters.measure {
-            val s = login.toJSON().toString()
```

Comment:
Congrats! You can now remove the `toJSON`/`fromJSON`/`fromJSONArray` methods from `ServerPassword` (I've checked there are no usages in consumers of application-services). Same in Swift with `toJSON()` (but keep `toJSONDict()`!).